akka-remote/src/main/resources/reference.conf

```diff
@@ -285,6 +285,11 @@ akka {
         flight-recorder {
           // FIXME it should be enabled by default, but there is some concurrency issue that crashes the JVM
           enabled = off
+          # Controls where the flight recorder file will be written. There are three options:
+          # 1. Empty: a file will be generated in the temporary directory of the OS
```

Comment:
Yeah, I was a bit unsure about that, currently it does log that flight recorder is enabled and the file location on transport start.